### ISSUE TITLE
Facebook hides posts while deleting indiscriminately - dangerous consequence

### ISSUE THREAD

**Issue by indiandragon** [weskerfoot/DeleteFB#88]
We already know Facebook employs counter measures while deleting posts indiscriminately (#39), but I am confident that it's not limited just to changing the layouts. Facebook actually hides individual posts at random(or with some pattern) while deleting linearly.

I had successfully deleted thousands of posts (partially using this tool & manually). Though it looked like I have deleted all the posts, every couple of months some random post becomes visible in my profile.

The likely reason is that the post was hidden from me when I was deleting (or) the worst case scenario is Facebook didn't delete the post when I deleted them although this is unlikely.

So incase you have deleted your posts, do visit your profile once in a while to see whether this is happening.

**Comment by weskerfoot** [comment]
Yeah, I've seen this. Sometimes old posts that you deleted will pop up again randomly. I'm not sure if there's a way to fix it other than regularly deleting.

**Comment by weskerfoot** [comment]
I think this will be remedied by switching to using the activity log for everything, although I'm actually not sure. They might do something similar to rate limit you there.

**Comment by ZeroOneLabs** [comment]
Sorry for the necro.

What do you think of "noise scrubbing" each post before deletion? What I mean by this, is editing the post, replacing the text with random strings, saving the post, and then deleting it. That way, even if FB restores the post, it has random gibberish. Maybe work in some ASCII art that spells out "DELETED"?

**Comment by lucasew** [comment]
Sorry for the necro ²

If this behavior happens only when you started deleting things you can fetch the list of all the posts before deleting them. The program will know all the post ids before start deleting something.

**Comment by weskerfoot** [comment]
> Sorry for the necro ²
> 
> If this behavior happens only when you started deleting things you can fetch the list of all the posts before deleting them. The program will know all the post ids before start deleting something.

the way it works now (with the mobile site) might not even have this issue, I can't confirm though
